clang/test/CIR/global-var-simple.cpp

```diff
@@ -1,5 +1,6 @@
 // Global variables of intergal types
 // RUN: %clang_cc1 -std=c++20 -triple x86_64-unknown-linux-gnu -fclangir -emit-cir %s -o %t.cir
+// RUN: cat %t.cir
```

Comment:
This looks like a mistake?

Reply:
Yes, it was for some testing, i removed it

Thanks